### ISSUE TITLE
Lack of symlinks for bluetooth input devices

### ISSUE THREAD

**Issue by hanetzer** [systemd/systemd#3072]
### Submission type
- [ ] Bug report
- [x] Request for enhancement (RFE)

_NOTE: Do not submit anything other than bug reports or RFEs via the issue tracker!_
### systemd version the issue has been seen with

> 229-3

_NOTE: Do not submit bug reports about anything but the two most recently released systemd versions upstream!_
### Used distribution

> Arch Linux x86_64
### In case of bug report: Expected behaviour you didn't see

> No symlinks for Sony Dualshock 4 over bluetooth under `/dev/input/by-{id,path}`
### In case of bug report: Steps to reproduce the problem

> Attach a Sony Dualshock 4 controller to your machine via bluetooth

Apparently caused by changes from this [bug report](https://bugzilla.redhat.com/show_bug.cgi?id=733862)

**Comment by poettering** [comment]
Isn't bluetooth using uinput to generate userspace input devices? If so, i doubt they are supposed to by "by-path" links at least. @kaysievers?

**Comment by hanetzer** [comment]
I don't think so. It can be used normally with the uinput kernel driver unloaded

**Comment by poettering** [comment]
Well, and it isn't autoloaded then?

Could you paste "udevadm info" on the input device?

**Comment by hanetzer** [comment]
on `/dev/input/event*` or `/dev/input/js*` ?

**Comment by hanetzer** [comment]
[`udevadm info /dev/input/event19`](https://gist.github.com/ntzrmtthihu777/b0f915afbcaea8e8e3739977824e41f6)

**Comment by hanetzer** [comment]
Nothing?

**Comment by poettering** [comment]
well, if these are indeed handled in the kernel, then i figure the "by-path" logic simply doesn't cover bluetooth devices right now. Not sure if it really should, but it probably should cover by-id stuff. Will mark this RFE since we never supported this.
